apps/asap-server/jest.setup.ts

```diff
@@ -0,0 +1,28 @@
+jest.mock('@asap-hub/auth', () => ({
+  decodeToken: jest.fn().mockResolvedValue({
```

Comment:
We already have a default mock for `@asap-hub/auth` with some things defined in `packages/auth/src/__mocks__/index.ts`. Can we add this in there instead? Then we also probably don't need to configure a setup file anymore because using it becomes a matter of `jest.mock('@asap-hub/auth')`